datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -907,7 +907,12 @@ type AspectRenderSpec {
 """
 Additional read only properties about a Dataset
 """
-type DatasetProperties {
+type DatasetProperties implements Aspect {
+    """
+    The logical version of the schema metadata, where zero represents the latest version
+    with otherwise monotonic ordering starting at one
+    """
+    version: Long
```

Comment:
is there a way to get this version field automatically by extending an interface called `VersionedAspect`?

Reply:
Unfortunately GraphQL interfaces do not push down fields, they just enforce them. GraphQL schemas are not designed around code reuse 😞